test/common/http/conn_manager_impl_test.cc

```diff
@@ -3121,6 +3121,135 @@ TEST_F(HttpConnectionManagerImplTest, FilterAddBodyContinuation) {
   encoder_filters_[1]->callbacks_->continueEncoding();
 }
 
+TEST_F(HttpConnectionManagerImplTest, AddDataWithAllContinue) {
```

Comment:
Given the complexity of this issue can you add test comments for this test and the test below describing what we are testing and why?

Reply:
Done